sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -961,35 +961,14 @@ static const zeCommandListBatchConfig ZeCommandListBatchCopyConfig = [] {
   return ZeCommandListBatchConfig(IsCopy{true});
 }();
 
-// Temporarily check whether immediate command list env var has been set. This
-// affects default behavior of make_queue API.
-static const bool ImmediateCommandlistEnvVarIsSet = [] {
-  const char *UrRet = std::getenv("UR_L0_USE_IMMEDIATE_COMMANDLISTS");
-  const char *PiRet =
-      std::getenv("SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS");
```

Comment:
Why are you removing this control? I think we should retain it for those who want to opt-out from using imm by default.

Reply:
The env var control is not being removed. This was a duplicated env var check to keep different defaults for old and new binaries.

Reply:
The env var control is not being removed. This was an extra reading of the env var to use different defaults for old and new binaries.

Reply:
@smaslov-intel : this change is good. There were two places where we were checking for this control.

Reply:
OK then.